kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
-# Run the text based installer to see dependency info in screenshots
+
+# By default run the text based installer to see dependency info in screenshots
+<% if ENV['KS_GUI'] == '1' then "" else "text" end %>
```

Comment:
I just noticed that this won't work because you didn't use `<%=` at the front.

Also,
- Prefer just checking for the presence of the env var vs a specific value...much easier for the user.
- Prefer ternary operator over if/then/else for single line

```erb
<%= ENV['KS_GUI'] ? "" : "text" %>
```

Can you update here and then make a PR for master to fix that one as well?